students/ian_letourneau/Lesson03/slicing_lab.py

```diff
@@ -0,0 +1,99 @@
+## Ian Letourneau
+## 4/26/2018
+## A script with various sequencing functions
+
+def exchange_first_last(seq):
+    """A function to exchange the first and last entries in a sequence"""
+    middle = list(seq)
+    middle[0] = seq[-1]
+    middle[-1] = seq[0]
+
+    if type(seq) is str:
+        return "".join(middle)
+    elif type(seq) is tuple:
+        return tuple(middle)
+    else:
+        return middle
```

Comment:
you should be using slicing only for this assignment -> `seq[start:end:step]`

`seq[0]` is indexing

hint: if you do use slicing then you don't need to worry about the type, it will work for all 3